README.md

```diff
@@ -1 +1,148 @@
+  <ul>
+    <li><a href="https://youtu.be/wIzPMotxMCM">Lesson on CSS3</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Provides insight to the current stage on the home page
+- Displays 10-day power outage schedule for all 590 zones across the country
+- Allows filtering zones to more easily find the desired one
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+You can find a live demo of the project [here](https://power-outage-tracker-qzsq.onrender.com/)
+
+You can find a link to a walkthrough of the project [here](https://drive.google.com/file/d/10R9Ht9bBOEPiha4FgDrAP5aWyyyLCnUE/view?usp=sharing)
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: <br>
+**To get a proper code editor installed on your computer alternatively you can use a web browser**
+
+
+### Setup
+
+Clone this repository to your desired folder: <br>
+`git clone https://github.com/aamir-asaram/power-outage-tracker`
+
+### Install
+
+Install this project with: <br>
+`npm i`
+
+### Usage
+
+To run the project, install necessary dependencies by running `npm install` and `npm start` to open it on the server.
+### Run tests
+
+To run tests, run the following command: <br>
+`npm test`
+
+### Deployment
+
+You can deploy this project using:
+- Render
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+👤 **Aamir Asaram**
+
+- GitHub: [aamir-asaram](https://github.com/aamir-asaram)
+- Twitter: [@aaaaamir__](http://twitter.com/aaaaamir)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aamir-asaram/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Pinning zones to the top of the page
+- [ ] Editing number of days displayed in schedule
+- [ ] Displaying upcoming outages for each zone on the home page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/aamir-asaram/power-outage-tracker/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you liked this project, kindly give it a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to acknowledge the creator of the design this web application is based on - [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.

Reply:
I didn't notice that, thanks a lot for pointing it out @shella12 :smile: